cajita/unit_test/tstSparseIndexSpace.hpp

```diff
@@ -237,15 +236,29 @@ void testBlockSpace()
             }
 }
 
-void testSparseIndexSpaceFullInsert()
+void testSparseMapFullInsert()
 {
     constexpr int dim_n = 3;
     constexpr int size_tile_per_dim = 16;
     constexpr int size_per_dim = size_tile_per_dim * 4;
 
     std::array<int, dim_n> size( { size_per_dim, size_per_dim, size_per_dim } );
-    int capacity = size_per_dim * size_per_dim;
-    SparseIndexSpace<TEST_EXECSPACE> sis( size, capacity );
+    int pre_alloc_size = size_per_dim * size_per_dim;
+    // Create the global mesh
+    double cell_size = 0.1;
+    std::array<int, 3> global_num_cell = size;
+    std::array<double, 3> global_low_corner = { 1.2, 3.3, -2.8 };
+    std::array<double, 3> global_high_corner = {
+        global_low_corner[0] + cell_size * global_num_cell[0],
+        global_low_corner[1] + cell_size * global_num_cell[1],
+        global_low_corner[2] + cell_size * global_num_cell[2] };
+    auto global_mesh = createSparseGlobalMesh(
+        global_low_corner, global_high_corner, global_num_cell );
+
+    // Create Sparse Map
+    // SparseMap<TEST_EXECSPACE> sis( size, pre_alloc_size );
+    auto sis =
+        createSparseMap<double, TEST_EXECSPACE>( global_mesh, pre_alloc_size );
```

Comment:
Once you have updated this above per my comment the only template parameter needed will be `TEST_MEMSPACE`

Reply:
That's true. Sounds great!